src/JSTest.Runner/ObjectModel/Common/TestCase.ts

```diff
@@ -30,5 +36,51 @@ export class TestCase {
             .appendStr(this.Source);
 
         this.Id = hash.getGuid();
+
+        this.AttachmentGuid = null;
+        if (attachmentId) {
+            const attachmentHash = new Md5();
+            attachmentHash.appendStr(attachmentId);
+            this.AttachmentGuid = attachmentHash.getGuid();
+        }
+    }
+
+    /**
+     * Looks for files to upload under the specified folder using test case specific id.
+     * 
+     * @param attachmentsRootFolder Root folder of the attachments.
+     */
+    public getAttachments(attachmentsRootFolder: string): AttachmentSet[] {
```

Comment:
so this guy gets called by someone, returns an AttachmentSet, who in turn sets it to TestResult?

Reply:
That's right. I can keep it in ExecutionManager as well.

Reply:
I feel it should be in basetestframework who's responsible for creating the test result object

Reply:
Ok, I'll move it.